docs/options/ncmoe.md

```
@@ -0,0 +1,6 @@
+####> This option file is used in:
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)